dev/start-local.sh

```diff
@@ -66,6 +67,8 @@ $SPARK_HOME/bin/spark-submit --class streaming.core.StreamingApp \
         --conf "spark.kryoserializer.buffer.max=1024m" \
         --conf "spark.serializer=org.apache.spark.serializer.KryoSerializer" \
         --conf "spark.scheduler.mode=FAIR" \
+        --conf "spark.driver.extraClassPath=${EXT_JARS}" \
+        --conf "spark.executor.extraClassPath=${EXT_JARS}" \
```

Comment:
This line requires that all jars should be placed in local disk path `${MLSQL_HOME}/libs`  in every executor. It will throw exception when in yarn/k8s mode.